src/pages/recipes/{ContentfulRecipe.title}.tsx

```diff
@@ -1,74 +1,105 @@
+          marginBottom={5}
+        >
+          {recipe.title}
+        </Heading>
+        <Flex direction={{ base: 'column-reverse', md: 'row' }} justify="space-between">
+          <VStack
+            align="start"
+            spacing={5}
+            marginTop={{ base: 5, md: 0 }}
+            marginRight={{ base: 0, md: 10 }}
           >
-            Ingredients
-          </h2>
-          <p
-            style={{
-              display: 'block',
-            }}
-          >
-            {recipe?.ingredients?.ingredients}
-          </p>
-          <h2
-            style={{
-              fontWeight: 'bold',
-            }}
-          >
-            Preparation Directions
-          </h2>
-          <p
-            style={{
-              display: 'block',
-            }}
-          >
-            {recipe?.prepDirections?.prepDirections}
-          </p>
-          <h2
-            style={{
-              fontWeight: 'bold',
-            }}
-          >
-            Cooking Directions
-          </h2>
-          <p
-            style={{
-              display: 'block',
-            }}
-          >
-            {recipe?.directions?.directions}
-          </p>
-        </div>
-      </div>
+            <HStack align="start" spacing={10}>
+              <VStack align="start">
+                <Text fontWeight="bold">Cook Time</Text>
+                <Text fontWeight="bold">Prep Time</Text>
+                <Text fontWeight="bold">Total Time</Text>
+                <Text fontWeight="bold">Yield</Text>
+                <Text fontWeight="bold">Ingredients</Text>
+              </VStack>
+              <VStack align="start">
+                <Text>{recipe.cookTime} min</Text>
+                <Text>{recipe.prepTime} min</Text>
+                <Text>{recipe.totalTime} min</Text>
+                <Text>{recipe.yield}</Text>
+                <VStack align="stretch">
+                  {recipe.ingredientGroups.map(group => (
+                    <Flex justify="space-between" key={group[0]}>
+                      <Box flex={2}>
+                        <Text as="span" display={{ base: 'inline-block', md: 'none' }}>
+                          {group[1]}
+                        </Text>{' '}
+                        <Text as="span" textTransform={{ base: 'lowercase', md: 'none' }}>
+                          {group[0]}
+                        </Text>{' '}
+                        {group[2] && (
+                          <Text as="span" color="gray">
+                            ({group[2]})
+                          </Text>
+                        )}
+                      </Box>
+                      <Box display={{ base: 'none', md: 'block' }} flex={1} marginLeft={10}>
+                        <Text>{group[1]}</Text>
+                      </Box>
+                    </Flex>
+                  ))}
+                </VStack>
+              </VStack>
+            </HStack>
+          </VStack>
+          <Image
```

Comment:
Rather than use Chakra's `Image`, we might want to use Gatsby's `gastby-image` (see https://www.gatsbyjs.com/plugins/gatsby-image/). We can get a `fluid` image and nest it in a Chakra `Box` to handle responsive sizing

(For now, ignore the deprecation warning - I'll likely run their suggested codemod at a later time)